setup.py

```diff
@@ -10,7 +10,7 @@
 import versioneer
 
 setup(
-    name='q2-diversity-lib',
+    name="q2-diversity-lib",
```

Comment:
TODO: let's add a note in the commit message that mentions all of these stylistic changes (they'll make the `git blame` a little confusing, but such is life).

Reply:
Added to commit-message comment. Good call!